playbooks/roles/openvpn/defaults/main.yml

```diff
@@ -13,6 +13,9 @@ openvpn_clients:
 
 openvpn_port: "636"
 openvpn_port_udp: "8757"
+openvpn_port_sslh: "443"
 openvpn_key_size: "2048"
-openvpn_cipher: "AES-256-CBC"
+openvpn_cipher: "AES-256-GCM"
+openvpn_ncp_cipher: "AES-256-GCM:AES-256-CBC"
 openvpn_auth_digest: "SHA256"
+openvpn_connect_timeout: 15 #seconds
```

Comment:
I think we could safely drop this down to 5 seconds. What do you think? Fifteen seconds is way better than the default, but it could potentially mean waiting 30 seconds (instead of 10) before the sslh connection option is attempted.